package.json

```diff
@@ -170,7 +175,10 @@
     "dev:webui": "cross-env BABEL_ENV=ui babel-node tools/dev.server.js",
     "build:webui": "npm run pre:webpack && cross-env BABEL_ENV=ui webpack --config tools/webpack.prod.config.babel.js",
     "build:docker": "docker build -t verdaccio . --no-cache",
-    "build:docker:rpi": "docker build -f Dockerfile.rpi -t verdaccio:rpi ."
+    "build:docker:rpi": "docker build -f Dockerfile.rpi -t verdaccio:rpi .",
+    "prettier-list": "prettier -l --config .prettierrc ./{test,src,tools}/**/*.js",
+    "prettier-write": "prettier --write --config .prettierrc ./{test,src,tools}/**/*.js",
```

Comment:
Using a prettier rc specific file don't make an integration between eslint and prettier. In fact, keeps them separately, not using the rules defined on eslint rc file

Reply:
yes..it's true..I just the same description you used before ;)